cafebabel/templates/base.html

```diff
@@ -1,8 +1,40 @@
+{% from "_meta-macro.html" import meta with context %}
```

Comment:
I don't think you need the `context` here?

Reply:
fixed